project_window/project_window.py

```diff
@@ -478,7 +479,7 @@ def retry_with_summary(self, summary):
             None
         )
         self.bottom_stack.preview_text.clear()
-        self.bottom_stack.preview_text.setReadOnly(True) # User clicks can really mess up the text inserts
+        self.bottom_stack.psetRe # User clicks can really mess up the text inserts
```

Comment:
This line appears to have been accidentally edited and needs to be reverted back to its original version.
```suggestion
        self.bottom_stack.preview_text.setReadOnly(True) # User clicks can really mess up the text inserts
```